src/main/java/thePackmaster/powers/cthulhupack/SanityPower.java

```diff
@@ -0,0 +1,68 @@
+package thePackmaster.powers.cthulhupack;
+
+import com.megacrit.cardcrawl.actions.common.LoseHPAction;
+import com.megacrit.cardcrawl.actions.common.MakeTempCardInHandAction;
+import com.megacrit.cardcrawl.actions.common.ReducePowerAction;
+import com.megacrit.cardcrawl.actions.watcher.ChangeStanceAction;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.cards.colorless.Madness;
+import com.megacrit.cardcrawl.core.AbstractCreature;
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import thePackmaster.cards.cthulhupack.Lunacy;
+import thePackmaster.powers.AbstractPackmasterPower;
+import thePackmaster.stances.cthulhupack.NightmareStance;
+import thePackmaster.util.Wiz;
+
+import java.util.Objects;
+
+import static thePackmaster.SpireAnniversary5Mod.makeID;
+
+public class SanityPower extends AbstractPackmasterPower {
+    public static final String POWER_ID = makeID("SanityPower");
+    public static final String NAME = CardCrawlGame.languagePack.getPowerStrings(POWER_ID).NAME;
+    public static final String[] DESCRIPTIONS = CardCrawlGame.languagePack.getPowerStrings(POWER_ID).DESCRIPTIONS;
+
+    public SanityPower(AbstractCreature owner, int amount) {
+        super(POWER_ID, NAME, PowerType.DEBUFF, false, owner, amount);
+        canGoNegative = true;
+    }
+
+    @Override
+    public void atStartOfTurnPostDraw() {
+        //The power string reads "if you have negative sanity" for flavor.  Since there is no positive sanity,
+        //Amount check is technically unnecessary but who knows, maybe someone down the road wants to make a positive sanity pack.
+        if (amount < 0) {
+            this.fontScale = 8.0F;
+            this.amount -= 1;
+            updateDescription();
+            AbstractDungeon.onModifyPower();
+        }
+    }
+
+    @Override
+    public void stackPower(int stackAmount) {
+        this.fontScale = 8.0F;
+        this.amount += stackAmount;
+        while (amount <= -5) {
+            amount += 5;
```

Comment:
The only reason I left it here is because when in use with the pack in general, you'll more often than not go to 0, and then reduce it more before the turn is done by playing the Lunacy you get.  This setup makes the power icon stay in place in that case instead of having it shift to the right again.  A minor convenience but will get rid of it if it's a problem.

Reply:
The power icon makes sense as a reason -- minor and only aesthetic, but I get it and it's fine with me.